### ISSUE TITLE
vbraun branch on Prefix will not build

### ISSUE THREAD

**Issue by strogdon** [cschwan/sage-on-gentoo#601]
I was investigating [ticket #30676](https://trac.sagemath.org/ticket/30676) by building the s-o-g current vbraun branch. The branch does build on Gentoo and I see the failure; however, the branch will not builld on Prefix. The failure
```
changing mode of /storage/strogdon/gentoo-rap/var/tmp/portage/sci-mathematics/sage-9999/work/sage-9999/src-python3_8/build/scripts/sage-run-cython from 644 to 755
changing mode of /storage/strogdon/gentoo-rap/var/tmp/portage/sci-mathematics/sage-9999/work/sage-9999/src-python3_8/build/scripts/sage-startuptime.py from 644 to 755
 ^[[32;01m*^[[0m Using python3.8 in global scope
 ^[[32;01m*^[[0m python3_8: running distutils-r1_run_phase python_compile_all
Traceback (most recent call last):
  File "sage_setup/docbuild/__main__.py", line 1, in <module>
    from sage_setup.docbuild import main
  File "/storage/strogdon/gentoo-rap/var/tmp/portage/sci-mathematics/sage-9999/work/sage-9999/src-python3_8/sage_setup/docbuild/__init__.py", line 58, in <module>
    import sage.all
  File "/storage/strogdon/gentoo-rap/var/tmp/portage/sci-mathematics/sage-9999/work/sage-9999/src-python3_8/build/lib/sage/all.py", line 129, in <module>
    from sage.data_structures.all import *
  File "/storage/strogdon/gentoo-rap/var/tmp/portage/sci-mathematics/sage-9999/work/sage-9999/src-python3_8/build/lib/sage/data_structures/all.py", line 3, in <module>
    from .bitset import Bitset, FrozenBitset
ImportError: /storage/strogdon/gentoo-rap/var/tmp/portage/sci-mathematics/sage-9999/work/sage-9999/src-python3_8/build/lib/sage/data_structures/bitset.cpython-38-x86_64-linux-gnu.so: undefined symbol: _bitset_intersection
 ^[[31;01m*^[[0m ERROR: sci-mathematics/sage-9999::sage-on-gentoo failed (compile phase):
```
I haven't investigated further since I have been away most of the day.

**Comment by kiwifb** [comment]
OK I have been working on that in https://trac.sagemath.org/ticket/30675 and I should update the patch in the vbraun branch in the meantime. The current patch only fixes three functions that were undefined for me. But it was always suspicious that none of the other were affected. In the ticket we decided to touch every single functions.

**Comment by kiwifb** [comment]
Fixed by 1bc7cbf5606f99fbb6714ab635c3015d23827a20
